qa-theme/Classic/qa-styles.css

```diff
@@ -474,3 +474,7 @@ h2 {font-size:16px; padding-top:12px; clear:both;}
 	0% { background-color: #ffffaa; }
 	100% { background-color: #fff; }
 }
+
+[dir="auto"] code {
```

Comment:
Please also add this style to the other themes. (We're already using *SnowFlat* theme at Divar)

Reply:
Done.